DirectProgramming/C++SYCL_FPGA/Tutorials/Features/private_copies/README.md

```diff
@@ -188,7 +196,12 @@ On the main report page, scroll down to the section titled "Estimated Resource U
      ./private_copies.fpga_emu     (Linux)
      private_copies.fpga_emu.exe   (Windows)
      ```
-2. Run the sample on the FPGA device:
+2. Run the sample on the FPGA simulator device:
+     ```
```

Comment:
With the new selectors, one must run the sim executable with a specific env variable.
See the instructions in another sample (e.g. https://github.com/oneapi-src/oneAPI-samples/tree/development/DirectProgramming/C%2B%2BSYCL_FPGA/Tutorials/GettingStarted/fpga_compile) and update the instructions here